File: DDDTableTopFriend.Domain/Entities/Character.cs
namespace DDDTableTopFriend.Domain.Entities;

public class Character : Entity
{
    public Guid UserId { get; set; }
    public Guid? CharacterSheetId { get; set; }
    public Guid? CampaignId { get; set; }
    public string Name { get; set; } = null!;
    public CharacterType Type { get; set; } = CharacterType.Player;
    CharacterSheet? CharacterSheet { get; set;}
}

File: TableTopFriendWebApp/src/app/core/models/character.ts
import { CharacterType } from "../constants/characterType";
import { CharacterSheet } from "./characterSheet";

export interface Character {
  userId: string,
  name: string,
  description: string,
  image: string,
  type: CharacterType,
  characterSheet: CharacterSheet
  audioEffectIds: string[]
}
